v2/azure-arm.yaml

```diff
@@ -1230,6 +1230,31 @@ objectModelConfiguration:
       RedisEnterprise_Database:
         $exportAs: RedisEnterpriseDatabase
         $supportedFrom: v2.3.0
+    2023-08-01:
+      Redis:
+        $export: true
+        $supportedFrom: v2.0.0-alpha.4
+        $azureGeneratedSecrets:
+          - PrimaryKey
+          - SecondaryKey
+          - HostName
+          - Port
+          - SSLPort
+      RedisCreateProperties:
```

Comment:
This is commented out in the other API versions? Do you know why?

Reply:
Not sure, but generator enforced me to add it in this time. Although looks like the shape looks the same in older API versions too.